lib/locreator.js

```diff
@@ -75,6 +75,12 @@ Locreator.prototype.addStarMethods = function locreator(locatorId, _locator, par
   locatorObject[locatorId + 'OptionValue'] = function (optionValue) {
     return drivex.selectByOptionValue(locator, optionValue, parentWebElement);
   };
+  locatorObject[locatorId + 'ValidateText'] = function (ValidateText) {
+    return drivex.validateText(locator, parentWebElement,ValidateText) ;
+  };
+  locatorObject[locatorId + 'ValidateAttributeValue'] = function (attribute,validateText) {
```

Comment:
can you please change the name of the parameter  from `validateText` to `comparisonValue` or `value` or something else which succinctly describes the purpose?